modules/sdk-coin-avaxp/src/avaxp.ts

```diff
@@ -163,14 +180,19 @@ export class AvaxP extends BaseCoin {
         break;
       case TransactionType.Import:
         if (tx.isTransactionForCChain) {
+          // ImportInC
           if (
             (params.txParams.recipients && params.txParams.recipients.length !== 0) ||
             explainedTx.outputs.length !== 1
           ) {
             throw new Error('Expected 1 output in import txn and does not require recipients');
           }
         } else {
-          // TODO: BG-55595 - verify import txn on P chain
+          // ImportInP
+          if (explainedTx.outputs.length !== 1) {
+            throw new Error('Expected 1 output in import txn');
+          }
+          this.validateImportInP(explainedTx.inputs, params.txParams);
```

Comment:
consistency-nit:
Please avoid using export "in" and import "in"
```suggestion
          this.validateImportTx(explainedTx.inputs, params.txParams);
```